test/Jsr292/src/com/ibm/j9/jsr292/LookupAPITests_Find.java

```diff
@@ -2359,6 +2359,37 @@ public void test_FindStatic_InitCheck() throws Throwable {
 		}
 	    AssertJUnit.assertTrue(NoSuchMethodExceptionThrown);
 	}
+	
+	/**
+	 * Test getVirtual on a concrete method inherited from Object returns a valid method handle.
+	 * @throws Throwable
+	 */
+	@Test(groups = { "level.extended" })
+	public void test_findVirtual_on_interface_concreteMethods() throws Throwable {
+		MethodHandle mh = MethodHandles.lookup().findVirtual(TestInterface.class,
+				"hashCode", MethodType.methodType(int.class));
+		mh.invokeExact(new TestInterface() {
```

Comment:
an MH on `hashCode` should return a `int`.  This `invokeExact` will fail with a WrongMethodTypeException at runtime.

Try `int hash = (int)mh.invokeExact(new TestInterface() {...});`